best-practices.md

```diff
@@ -2,20 +2,24 @@
 
 ## Table of Contents
 
-* [Enable CORS](#enable-cross-origin-resource-sharing-cors)
-* [Field and ID formatting](#field-and-id-formatting)
-* [Field selection and Metadata Linking](#field-selection-and-metadata-linking)
-* [Datetime selection](#datetime-selection)
-* [Unlocated Items](#unlocated-items)
-* [Representing Vector Layers in STAC](#representing-vector-layers-in-stac)
-* [Common Use Cases of Additional Fields for Assets](#common-use-cases-of-additional-fields-for-assets)
-* [Working with Media Types](#working-with-media-types)
-* [Static and Dynamic Catalogs](#static-and-dynamic-catalogs)
-* [Catalog Layout](#catalog-layout)
-* [Use of Links](#use-of-links)
-* [Using Relation Types](#using-relation-types)
-* [Versioning for Catalogs](#versioning-for-catalogs)
-* [STAC on the Web](#stac-on-the-web)
+* **[Web Best Practices](#web-practices)**
+  * [Enable CORS](#enable-cross-origin-resource-sharing-cors)
+  * [STAC on the Web](#stac-on-the-web)
+* **[Item Best Practices](#item-practices)**
+  * [Field and ID formatting](#field-and-id-formatting)
+  * [Field selection and Metadata Linking](#field-selection-and-metadata-linking)
+  * [Datetime selection](#datetime-selection)
+  * [Unlocated Items](#unlocated-items)
+  * [Representing Vector Layers in STAC](#representing-vector-layers-in-stac)
+* **[Asset Best Practices](#asset-practices)**
+  * [Common Use Cases of Additional Fields for Assets](#common-use-cases-of-additional-fields-for-assets)
+  * [Working with Media Types](#working-with-media-types)
```

Comment:
Also applies to links?!

Reply:
It does, but that whole section is really focused on asset media types. I can rename the title to 'working with asset media types'. Or could move it, but not sure where - I'm mostly looking for the best way to 'group' things to help people navigate a bit. Links cuts across all of them...